nixos/modules/services/misc/conduit.nix

```
@@ -0,0 +1,118 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.conduit;
+
+  format = pkgs.formats.toml {};
+  namespacedSettings.global = cfg.settings;
+  configFile = format.generate "conduit.toml" namespacedSettings;
```

```suggestion
  configFile = format.generate "conduit.toml" cfg.settings;
```

What happens if something besides the global section is added? I would recommend against going this route.